nlg/grammar.py

```diff
@@ -237,10 +237,11 @@ def find_inflections(text, search, fh_args, df):
         tmpl = [t['tmpl'] for t in tklist if t.get('enabled', False)][0]
         rendered = Template('{{{{ {} }}}}'.format(tmpl)).generate(
             df=df, fh_args=fh_args).decode('utf8')
-        if rendered != token:
+        if rendered != getattr(token, "text", token):
```

Comment:
Quants and fh_args are still not ported over to spacy tokens and spans - which has necessitated this.